graphrag/config/create_graphrag_config.py

```diff
@@ -522,8 +525,12 @@ def hydrate_parallelization_params(
             )
 
         with reader.use(values.get("cluster_graph")):
+            use_lcc = reader.bool("use_lcc")
             cluster_graph_model = ClusterGraphConfig(
-                max_cluster_size=reader.int("max_cluster_size") or defs.MAX_CLUSTER_SIZE
+                max_cluster_size=reader.int("max_cluster_size")
+                or defs.MAX_CLUSTER_SIZE,
```

Comment:
Can the default value here be defined in the config pydantic model directly instead?

Reply:
It is set in the pydantic model as well. I'm not yet sure how we will remove the redundancy, but we need to do it everywhere. Hoping Derek has a good solution